### ISSUE TITLE
[BUG] Folder with "/" doesnt sync proper

### ISSUE THREAD

**Issue by zft9xgy** [FilenCloudDienste/filen-desktop#234]
Desktop version v2.0.24
MacOs Ventura 13.5.2 

Case
1. USB connect to save a local backup on filen. 
2. Create a folder in the coud ex. "Folder in the cloud"
3. Set up the "local backup" sync from "my local folder" to "folder in the cloud"
4. Inside "my local folder" i got a folder with "photos 2020/2021" with plenty of photos. 
5. After the sync is done, when i acces to filen.io web version, the folder is name as "photos 2020-2021" and is empty. 

So basically it seem like it got conflict with certain caracter in folder name, in this case "/" it is replace for a "-" and then it detect as a different folder and dont backup the data inside. 

Another question to considered. 
Could be different caracter which present the same conflict?
Is it also affect file name? 

If you need more data or logs, im open to colaborate.

**Comment by rburettt** [comment]
Can someone please look at this issue? I encountered the same problem yesterday

**Comment by Dwynr** [comment]
This is currently by design since we support Windows, macOS and various Linux distros with almost the same codebase.

One way to fix this would be to modify the function which checks if a file/folder name has valid characters. The list of valid chars needs to be different for each operating system. You are currently encountering this issue because the current list of valid chars is Windows based.

I will look into fixing it in the next release.
